src/neighborhoods/ball.jl

```diff
@@ -37,6 +37,21 @@ metric(ball::BallNeighborhood) = ball.metric
 isneighbor(ball::BallNeighborhood, xₒ::AbstractVector, x::AbstractVector) =
   evaluate(ball.metric, xₒ, x) ≤ ball.radius
 
+"""
+    EllipsoidNeighborhood(semiaxes::AbstractVector, angles::AbstractVector;
+      convention::Symbol=:TaitBryanExtr)
+
+An ellipsoid neighborhood with `semiaxes` and `angles`. For 2D ellipses,
+there are two semiaxes and one rotation angle. For 3D ellipsoids, there are
+three semiaxes and three rotation angles. Different rotation conventions can be
+passed via the `convention` keyword argument. The list of conventions is
+available in the `aniso2distance` documentation.
```

Comment:
Awesome docstring. You can use `[`aniso2distance`](@ref)` to make it a link in the docs.

Reply:
GitHub is messing up with the single quotes. Check other example of `@ref` in the codebase for links in docstrings 👌🏽

Reply:
Nice trick, I didn't know that